databricks/koalas/missing/window.py

```diff
@@ -71,14 +67,10 @@ class _MissingPandasLikeRolling(object):
     count = unsupported_function_rolling("count")
     cov = unsupported_function_rolling("cov")
     kurt = unsupported_function_rolling("kurt")
-    max = unsupported_function_rolling("max")
-    mean = unsupported_function_rolling("mean")
     median = unsupported_function_rolling("median")
-    min = unsupported_function_rolling("min")
     quantile = unsupported_function_rolling("quantile")
     skew = unsupported_function_rolling("skew")
     std = unsupported_function_rolling("std")
-    sum = unsupported_function_rolling("sum")
```

Comment:
maybe you only implemented that functions for `Expanding`??

Reply:
Yeah, it's for simplicity. `ExpandingGroupBy` extends `Expanding` so it inherits the implementation. As a workaround for now, I am manually throwing an exception at `ExpandingGroupBy`.

This missing methods are tested so this is just to make the tests pass for now.

Reply:
@HyukjinKwon Thanks for the comment!

okay i got it, but i mentioned about these commented lines are indicates functions of `Rolling`, not `Expanding` 😄

Reply:
Oh, you're right. let me check.